mergerequest.html

```diff
@@ -66,31 +66,26 @@
 
 --
 .answer[
-- Code review
+- Spread knowledge across the team
 - Increase the quality of the product
 - Collaboration and Learning
 - Identify design flaws
-- Ensure code reuse (does this function already exist ?)
-- Improve knowledge of the solutions components and what others are working on and how it was implemented.
+- Code reusability (does function already exist ?)
 ]
```

Comment:
This sounds like the "goal of a Code Review".
The "goal of a Merge/Pull Request" is more about enforcing policies on all changed code including: code review, associated to approved work, passing a series of checks (though a pipeline). _(I'm probably missing something)._